INSTALL-XAMPP.md

```diff
@@ -8,7 +8,7 @@ XAMPP.
 
  1. Install [XAMPP](https://www.apachefriends.org/), e.g. in the folder `C:\xampp\`
  2. Clone malibu in the `htdocs` subfolder
- 3. Download [jQuery](https://code.jquery.com/jquery-3.2.1.min.js) and place it in `.\htdocs\malibu\isbn`
+ 3. Download [jQuery](https://code.jquery.com/jquery-3.2.1.min.js) and [clipboard.js 1.7.1](https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js) and place them in `.\htdocs\malibu\isbn`
```

Comment:
Replace `clipboard.js` by `clipboard.min.js` here.

Reply:
Well, the names of the libraries are jQuery and clipboard.js, and both of them should be downloaded in the minified version... Isn't it more unclear if we only write the minified version of clipboard but not jQuery?

Reply:
You are right. I did not know that clipboard.js is the product name.